pixel/settings.py

```diff
@@ -152,15 +154,41 @@ class Development(Base):
         "SHOW_TOOLBAR_CALLBACK": lambda request: True,
     }
 
+    EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+
 
 class Staging(Base):
+    """
+    Depends on environment variables that SHOULD be defined (in addition to the
+    base environment variables):
+
+    EMAIL_HOST=smtp.example.org
+    EMAIL_PORT=587
+    EMAIL_HOST_USER=babar
+    EMAIL_HOST_PASSWORD=KingOfTheElephants
+    """
 
     ALLOWED_HOSTS = ['staging.pixel.candihub.eu', ]
 
+    EMAIL_HOST = values.Value(
+        '', environ_name='EMAIL_HOST', environ_prefix=None)
```

Comment:
Syntactically speaking, I prefer when the closing bracket position matches the block it closes, _e.g._:

```python
EMAIL_HOST = values.Value(
    '', environ_name='EMAIL_HOST', environ_prefix=None
)
```